Resources/Computing/ARCComputingElement.py

```diff
@@ -423,6 +423,13 @@ def getJobOutput( self, jobID, localDir = None ):
 
     isItOkay = job.Retrieve( self.usercfg, arc.URL( workingDirectory ), False )
     if ( isItOkay ):
+      try:
+        with open(outFileName, 'r') as outFile:
+          output = outFile.read()
+      except IOError as e:
+        return S_ERROR("Error downloading outputs", repr(e))
+      # if not os.path.isfile(outFileName):
+      #   return S_ERROR("Error downloading outputs - log file %s does not exist." % outFileName)
       outFile = open( outFileName, 'r' )
```

Comment:
But then, if all is well, outFile will not be defined? I am not sure but I am not a python expert.